src/main/webapp/common.js

```diff
@@ -117,30 +117,49 @@ async function displayCurrentAddress() {
  * longitude input elements will be disabled.
  */
 function updateStartAddress(geolocationButtonId, addressId) {
-  if (document.getElementById(geolocationButtonId).checked) {
-    // update the address box to show the address found using Geolocation
-    if (navigator.geolocation) {
-        // if location was retrieved successfully, call setLatLng, else alert in geolocationError
-        getLocation().then((position) => {
-          setLatLng(position);
-          getAddressFromLatLng(addressId)
-        }).catch((error) => geolocationError(error));
-    } else {
-      // clear the address box if location is not supported 
-      document.getElementById(addressId).value = ''; 
-      alert('Geolocation is not supported by this browser!');
-    }
+  geolocationButton = document.getElementById(geolocationButtonId);
+  addressBox = document.getElementById(addressId);
+  if (geolocationButton.checked) {
+    updateAddressBoxUsingGeolocation(addressBox);
+  } else {
+    // Allow the user to set the address.
+    addressBox.value = ''; 
+    disableCoordinates(document.getElementById('latitude'),
+      document.getElementById('longitude'));
+  }
+}
+
+/**
+ * Disables latitude and longitude hidden input elements.
+ */
+function disableCoordinates(lat, lng) {
+  lat.disabled = true;
+  lng.disabled = true;
+}
```

Comment:
This confused me for a second - it is common practice in frontend code to use `Elem` suffix to indicate that we're talking about a DOM element.

```suggestion
function disableCoordinates(latElem, lngElem) {
  latElem.disabled = true;
  lngElem.disabled = true;
}
```

Reply:
Thanks, I changed it.